Documentation/ceph-kms.md

```diff
@@ -142,12 +142,14 @@ security:
     connectionDetails:
         KMS_PROVIDER: vault
         VAULT_ADDR: https://vault.default.svc.cluster.local:8200
-        VAULT_BACKEND_PATH: rook/ver1
+        VAULT_BACKEND_PATH: rook
```

Comment:
Right, this happens because of https://github.com/rook/rook/pull/9528/files#diff-414567e67bd81e387dd942420465190203e1a1ad6e7e8840f731eb9e69f84775L185

With `vault secrets enable -path=rook kv`.

Reply:
I believe I've updated per the instructions.

Reply:
No the commit title is incorrect, just remove the top line, right now you have:

```
Correct VAULT_BACKEND_PATH in documentation
docs: Correct VAULT_BACKEND_PATH
```

Just remove `Correct VAULT_BACKEND_PATH in documentation`